### PR TITLE
Changes to allow compilation against the latest 4.0.0 snapshot

### PR DESCRIPTION
**Pull request by timyates**

### CODE REVIEW

File: test-suite/src/test/java/views/NonHTMLViewRendererTest.java [hunk 1]
```diff
@@ -168,7 +167,7 @@ public boolean exists(@NonNull String viewName) {
         }
     }
 
-    @Introspected
+    @Serdeable
```
Comment (timyates) [inline test-suite/src/test/java/views/NonHTMLViewRendererTest.java:170]: I do not know why I had to use `@Serdeable` in place of `@Introspected`...  Does 4.0.0 of security switch to serde?

File: views-core/build.gradle [hunk 2]
```diff
@@ -13,6 +13,7 @@ dependencies {
 
     implementation(libs.reactor.core)
 
+    testImplementation(mn.micronaut.serde.jackson)
```
Comment (sdelamo) [inline views-core/build.gradle:16]: please, use serde instead

**Review by sdelamo — CHANGES_REQUESTED** [review]
Please use serde instead jackson

File: test-suite/build.gradle [hunk 3]
```diff
@@ -21,7 +21,14 @@ dependencies {
     testImplementation(mn.micronaut.http.client)
     testRuntimeOnly(libs.logback.classic)
 
-    testImplementation(mn.micronaut.security)
+    testImplementation(mn.micronaut.security) {
+        // TODO: Remove once core is updated to use a snapshot prior to 4.0.0-M1
+        version {
+            strictly("4.0.0-SNAPSHOT")
```
Comment (sdelamo) [inline test-suite/build.gradle:27]: remove remove https://github.com/micronaut-projects/micronaut-core/pull/8298

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: test-suite/build.gradle (hunk 3 above)
Comment (timyates) [reply test-suite/build.gradle]: Done https://github.com/micronaut-projects/micronaut-views/pull/460/commits/9d496aaf0966d4c5c942082e69d2d00785f2fbde

**Review by sdelamo — APPROVED** [review]

**Pull request merged**